qiskit_ibm_runtime/options/resilience_options.py

```diff
@@ -0,0 +1,52 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Resilience options."""
+
+from typing import Optional, Sequence
+from dataclasses import dataclass
+from typing_extensions import Literal
+
+from .utils import _flexible
+
+NoiseAmplifierType = Literal[
+    "TwoQubitAmplifier",
+    "GlobalFoldingAmplifier",
+    "LocalFoldingAmplifier",
+    "CxAmplifier",
+]
+ExtrapolatorType = Literal[
+    "LinearExtrapolator",
+    "PolynomialExtrapolator",
```

Comment:
```suggestion
```

This one is redundant: the type upon which all others are based. It defaults to `LinearExtrapolator` when no extra configuration is given, so should be removed. I updated the documentation accordingly.